src/allmydata/storage_client.py

```diff
@@ -589,6 +634,10 @@ def from_announcement(cls, server_id, furl, ann, storage_server):
             short_description = server_id[:8]
         nickname = ann.get("nickname", "")
 
+        ## XXX FIXME post-merge, need to parse out
+        ## "grid-manager-certificates" and .. do something with them
+        ## (like store them on this object)
```

Comment:
What's going on here?  Stuff seems to basically work even though this hasn't been addressed.

Reply:
hmm, yes looks like just a wrong comment (perhaps from .. some merge resolution? or just leftover?)